exercises/concept-wip/arys-amazing-lasagna/.meta/config.json

```
@@ -0,0 +1,10 @@
+{
+  "authors": ["meatball133"],
+  "files": {
+    "solution": ["src/arys_amazing_lasagna.cr"],
+    "test": ["spec/arys_amazing_lasagna_spec.cr"],
+    "example": [".meta/src/example.cr"]
```

For concept exercises, we strive to design them such that there is only one idiomatic solution, which we call the _exemplar_. Maybe rename the file to match this?

```suggestion
    "example": [".meta/src/exemplar.cr"]
```